tests/test_helpers/dom_helper.ts

```diff
@@ -131,10 +131,44 @@ export function triggerMouseEvent(
   });
   (ev as any).offsetX = offsetX;
   (ev as any).offsetY = offsetY;
+  dispatchEvent(selector, ev);
+}
+
+export function triggerWheelEvent(
+  selector: string | EventTarget,
+  extra: WheelEventInit = {
+    bubbles: true,
+    deltaMode: WheelEvent.DOM_DELTA_PIXEL, // = 0
+    deltaX: 0,
+    deltaY: 0,
+  }
+) {
+  const ev = new WheelEvent("wheel", { bubbles: true, ...extra });
+  dispatchEvent(selector, ev);
+}
+
+export function triggerKeyDown(selector: string | EventTarget, eventArgs: KeyboardEventInit) {
```

Comment:
We could go one step further: `function triggerKeyDown(selector, key, eventArgs)`.
It will be easier to write `triggerKeyDown(document.body, "F2")`
Same for keyup obviously

Reply:
I thought about doing that, I ended up with the conculsion that `triggerKeyDown(document.body, "F2")` is easier to write, but I slightly prefer to read `triggerKeyDown(document.body, { key : "F2" })`, I like named arguments.

But I don't have a strong opinion on this, if people prefer the other I can change

Reply:
I like @pro-odoo 's suggestion as it is closer to the original API of dispathEvent.